Core/source/test.ts

```diff
@@ -349,28 +352,29 @@ ava.default('parseKeys', async t => {
           { "fingerprint": "9EF2F8F36A841C0D5FAB8B0F0BAB9C018B265D22", "longid": "0BAB9C018B265D22", "shortid": "8B265D22", "keywords": "ARM FRIEND ABOUT BIND GRAPE CATTLE" }
         ],
         "algo": {
-          "algorithm": "rsa_encrypt_sign",
-          "bits": 2048,
+          "algorithm": "rsaEncryptSign",
```

Comment:
This will be a breaking change for the iOS app (maybe). We'll see later if it breaks the test.

Reply:
They have changed those names, so I had to change them too.

Reply:
Got it. Just highlighting that someone may have to update Swift code after the code is deployed to the app, possibly.

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1379